keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/OrcaService.kt

```diff
@@ -43,16 +46,28 @@ interface OrcaService {
   ): TaskRefResponse
 
   @GET("/pipelines/{id}")
-  suspend fun getPipelineExecution(@Path("id") id: String): ExecutionDetailResponse
+  suspend fun getPipelineExecution(
+    @Path("id") id: String,
+    @Header("X-SPINNAKER-USER") user: String = DEFAULT_SERVICE_ACCOUNT
```

Comment:
I added these last on the list of arguments so we don't have to refactor a gazillion tests.